library/ziti.c

```diff
@@ -426,7 +426,12 @@ static void ziti_init_async(ziti_context ztx, void *data) {
     ZTX_LOG(INFO, "using uv_mbed[%s], tls[%s]", uv_mbed_version(), ztx->tlsCtx->api->version ? ztx->tlsCtx->api->version() : "unspecified");
     ZTX_LOG(INFO, "Loading from config[%s] controller[%s]", ztx->opts->config, ztx_controller(ztx));
 
-    ziti_ctrl_init(loop, &ztx->controller, ztx_controller(ztx), ztx->tlsCtx);
+    if (ziti_ctrl_init(loop, &ztx->controller, ztx_controller(ztx), ztx->tlsCtx) != ZITI_OK) {
+	ZITI_LOG(ERROR, "Ziti controller init failed");
+	uv_stop(loop);
```

Comment:
remove this -- the loop is not controlled by the ziti_context, it is provided by the code using SDK.

Reply:
I'm using sample_host for testing invalid config file. The app will be stuck in loop and do nothing if uv_stop don't call.
Could you please advise me how to properly propagate the error to control code? Or such behavior is sufficient in this case?

Reply:
without callback application won't receive the error

Reply:
Changed to ziti_send_event